examples/stomp/stomp_durable_topic_subscriber.bal

```diff
@@ -0,0 +1,39 @@
+// This is the service implementation for STOMP protocol.
+import ballerina/stomp;
+import ballerina/log;
+import ballerina/io;
```

Comment:
In services we use only log and in main we use only io in examples. Shall we adhere to that?